contrib/kind.sh

```diff
@@ -391,7 +391,7 @@ if [ "${GITHUB_ACTIONS:-false}" == "true" ]; then
       -e 's/^.*kubernetes cluster\.local/& net/' \
       -e '/^.*upstream$/d' \
       -e '/^.*fallthrough.*$/d' \
-      -e '/^.*forward . \/etc\/resolv.conf$/d' \
+      -e '/^.*forward . 8.8.8.8/d' \
```

Comment:
I'm not sure how this fixes it. This is just deleting an entry with 8.8.8.8 isnt it? So really you are just using /etc/resolv.conf now? This also only happens when github actions is running, so it wont fix running on your own laptop.

Reply:
try changing it to just be this for ipv4:
```
-e 's/^\(.*forward \.\).*$/\1 8.8.8.8 {/'
```

for ipv6: 2001:4860:4860::8888

Reply:
done